layersvt/api_dump.h

```diff
@@ -441,6 +441,11 @@ class ApiDumpSettings {
             stream() << "[\n";
         }
 
+        if (use_spaces)
+            stream() << std::setfill(' ');
+        else
+            stream() << std::setfill('\t');
+
```

Comment:
This seems unrelated to the commit comment.  It's also clearly a behavioral change; its correctness is unclear from context.  Please explain.

Reply:
Check the api dump refactor PR - this commit is in that and I address that issue there (with a comment)